x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/markdown/index.js

```diff
@@ -7,20 +7,48 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Markdown from 'markdown-it';
+import { getSecureRelForTarget } from '@elastic/eui';
 import { RendererStrings } from '../../../i18n';
 
 const { markdown: strings } = RendererStrings;
 
-const md = new Markdown();
+const md = new Markdown({ linkify: true });
 
 export const markdown = () => ({
   name: 'markdown',
   displayName: strings.getDisplayName(),
   help: strings.getHelpDescription(),
   reuseDomNode: true,
   render(domNode, config, handlers) {
-    const html = { __html: md.render(String(config.content)) };
     const fontStyle = config.font ? config.font.spec : {};
+    const openLinksInNewTab = config.openLinksInNewTab;
```

Comment:
Rather than copy all of this logic in here, we should `import {Markdown} from 'src/legacy/core_plugins/kibana_react/public/markdown'`

Then this method can just become something like 

```js
ReactDOM.render(
<Markdown className="canvasMarkdown" markdown={config.content} openLinksInNewtabs={config.openLinksInNewTab} />
```

Reply:
@crob611 I tried this import, but had to add to the method with an `else` statement, as using it as-is doesn't account for the default `undefined` state. 

Using the original method, if you toggle to `true`, the links will open in a new tab, but if you then toggle to `false` (with the original method), it persists the `true` value (links still opened in a new tab). 

With the modification; adding the the `else` statement, it accounts for the default (undefined), explicit `true`, and explicit `false` conditions. 

I'm not sure how best to append the condition to the imported method.

Reply:
@crob611 I've update the code per your help/recommendation, and got the text not to wrap, but looks like the toggle example from the bar chart is using axisConfig (when I specified argType of `axisConfig` instead of toggle, it looked just like the `Y-axis` section, but the `toggleArgInput` seems to need to be updated with different styling to render it the same way. I'm working to understand this a bit better.